stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -981,7 +987,7 @@ function rank(A::AbstractMatrix; atol::Real = 0.0, rtol::Real = (min(size(A)...)
     isempty(A) && return 0 # 0-dimensional case
     s = svdvals(A)
     tol = max(atol, rtol*s[1])
-    count(x -> x > tol, s)
+    count(>(tol), s)
```

Comment:
```suggestion
    return count(>(tol), s)
```
Might as well use the improved readability of an explicit `return` while we are editing this line.

Reply:
I prefer the style without `return` but because this is orthogonal both to the documentation change and to the code change I'm going to call it out of scope for this PR and leave it as is for now.

Reply:
I prefer `x -> x > tol`. That change you made also seem out of scope for this PR :)